src/Service/DateTimeHelper.php

```diff
@@ -52,6 +52,33 @@ public function getFirstAndLastDateOfMonth(int $monthNumber, int $year, string $
         return ['first' => $firstDate, 'last' => $lastDate];
     }
 
+    /**
+     * Calculate the number of weekdays (Mon-Fri) between two dates in an associative array.
+     *
+     * @param string $dateFrom
+     * @param string $dateTo
+     *
+     * @return int
+     *
+     * @throws \Exception
+     */
+    public function getWeekdaysBetween(string $dateFrom, string $dateTo): int
```

Comment:
Can the conversion to date be handled in an earlier step? So this function does not have to convert from string.